nest-forms-backend/src/ginis/subservices/ginis-api.service.ts

```diff
@@ -57,4 +63,24 @@ export default class GinisAPIService {
       'Id-osoby': functionDetail['Detail-funkcniho-mista']['Id-referenta'],
     })
   }
+
+  async uploadFile(
+    documentId: string,
+    fileName: string,
+    contentStream: Readable,
+  ): Promise<SslPridatSouborPridatSoubor> {
+    const baseName = path.parse(fileName).name
+
+    const fileUpload: SslPridatSouborResponse =
```

Comment:
This is autoderived, this is an antipattern in TS when you don't want it to match the type, please just:
```
const fileUpload =
```

Reply:
fixed in [Remove autoderived type in ginis api service](https://github.com/bratislava/konto.bratislava.sk/pull/2614/commits/b9a60f003c5fd1f8cf36db320564aede207f26df)